### PR TITLE
Grammar issues fixed across component descriptions. New images added for clarity.

### PR DESCRIPTION
**Pull request by attraylor**
These grammar changes were made as a part of App Inventor Clean Up Day.

### CODE REVIEW

File: appinventor/components/src/com/google/appinventor/components/runtime/Image.java [hunk 1]
```diff
@@ -37,7 +37,7 @@
 @DesignerComponent(version = YaVersion.IMAGE_COMPONENT_VERSION,
     category = ComponentCategory.USERINTERFACE,
     description = "Component for displaying images.  The picture to display, " +
-    "and other aspects of the Image's appearance, can be specified in the " +
+    "and other aspects of the image's appearance, can be specified in the " +
```
Comment (josmas) [inline appinventor/components/src/com/google/appinventor/components/runtime/Image.java]: I would leave this as **Image** with a capital as it is referring to the component (noun).

File: appinventor/components/src/com/google/appinventor/components/runtime/ListView.java [hunk 2]
```diff
@@ -190,7 +190,7 @@ public void Width(int width) {
   @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_BOOLEAN,
       defaultValue = DEFAULT_ENABLED ? "True" : "False")
   @SimpleProperty(description = "Sets visibility of ShowFilterBar. True will show the bar, " +
-      "False will hide it.")
+      "false will hide it.")
```
Comment (josmas) [inline appinventor/components/src/com/google/appinventor/components/runtime/ListView.java]: this is also a noun; it the value **False** with capital F that needs to be specified; as shown in line 191.

File: appinventor/components/src/com/google/appinventor/components/runtime/Slider.java [hunk 3]
```diff
@@ -264,7 +264,7 @@ public float MinValue() {
   @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_FLOAT,
       defaultValue = Component.SLIDER_MAX_VALUE + "")
   @SimpleProperty(description = "Sets the maximum value of slider.  Changing the maximum value " +
-      "also resets Thumbposition to be halfway between the minimum and the (new) maximum. " +
+      "also resets the thumb position to be halfway between the minimum and the (new) maximum. " +
```
Comment (josmas) [inline appinventor/components/src/com/google/appinventor/components/runtime/Slider.java]: ThumbPosition is a property; the only change needed here would be a capital P --> ` also resets ThumbPosition to be halfway...`

File: appinventor/components/src/com/google/appinventor/components/runtime/WebViewer.java [hunk 4]
```diff
@@ -384,7 +384,7 @@ public void UsesLocation(boolean uses) {
    * @return true if prompting is  required.  False assumes permission is granted.
    */
 
-  @SimpleProperty(description = "If True, then prompt the user of the WebView to give permission to access the geolocation API. " +
+  @SimpleProperty(description = "If true, then prompt the user of the WebView to give permission to access the geolocation API. " +
```
Comment (josmas) [inline appinventor/components/src/com/google/appinventor/components/runtime/WebViewer.java]: This should (in my opinion) stay with capital, but if you are going to change it, it should be consistent and change also False in line 388).

File: appinventor/docs/reference/components/userinterface.html [hunk 5]
```diff
@@ -640,7 +640,7 @@ <h2 id="DatePicker">DatePicker</h2>
                  <h2 id="Image">Image</h2>
 
                  <p>Component for displaying images.  The picture to
-                 display, and other aspects of the Image's appearance,
+                 display, and other aspects of the image's appearance,
```
Comment (josmas) [inline appinventor/docs/reference/components/userinterface.html]: Image is a noun; same as DatePicker.

File: appinventor/docs/reference/components/userinterface.html [hunk 6]
```diff
@@ -792,30 +792,30 @@ <h2 id="ListPicker">ListPicker</h2>
...
   <dt><code>Height</code></dt>
-  <dd>Determines the height of the list on the view.</dd>
+  <dd>Determines the height of the list on the view (y-size).</dd>
   <dt><code>Selection</code></dt>
   <dd>Returns the text last selected in the ListView.</dd>
   <dt><code>SelectionIndex</code></dt>
   <dd>The index of the currently selected item, starting at 1.  If no item is selected, the value will be 0.  If an attempt is made to set this to a number less than 1 or greater than the number of items in the ListView, SelectionIndex will be set to 0, and Selection will be set to the empty text.</dd>
   <dt><code>ShowFilterBar</code></dt>
-  <dd>Sets visibility of ShowFilterBar. True will show the bar, False will hide it.</dd>
+  <dd>Sets visibility of the filter bar. True will show the bar, false will hide it.</dd>
```
Comment (josmas) [inline appinventor/docs/reference/components/userinterface.html]: ShowFilterBar as a noun, and also False as a noun (value). I would not change these.

File: appinventor/docs/reference/components/userinterface.html [hunk 7]
```diff
@@ -1003,11 +1004,13 @@ <h3 id="sliderproperties">Properties</h3>
 <dl><dt><code>ColorLeft</code></dt>
 <dd>The color of slider to the left of the thumb.</dd>
 <dt><code>ColorRight</code></dt>
-<dd>The color of slider to the left of the thumb.</dd>
+<dd>The color of slider to the right of the thumb.</dd>
+<dt><code>Height</code></dt>
+<dd>Height of the slider (y-size).</dd>
 <dt><code>MaxValue</code></dt>
-<dd>Sets the maximum value of slider.  Changing the maximum value also resets Thumbposition to be halfway between the minimum and the (new) maximum. If the new maximum is less than the current minimum, then minimum and maximum will both be set to this value.  Setting MaxValue resets the thumb position to halfway between MinValue and MaxValue and signals the PositionChanged event.</dd>
+<dd>Sets the maximum value of slider. Changing the maximum value also resets the thumb position to be halfway between the minimum and the (new) maximum. If the new maximum is less than the current minimum, then minimum and maximum will both be set to this value.  Setting MaxValue resets the thumb position to halfway between MinValue and MaxValue and signals the PositionChanged event.</dd>
```
Comment (josmas) [inline appinventor/docs/reference/components/userinterface.html]: ThumbPosition (as above) also in line 1013.

File: appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java [hunk 8]
```diff
@@ -5177,7 +5177,7 @@ String newerVersionComponentException(String componentType, int srcCompVersion,
   @Description("")
   String HorizontalArrangementHelpStringComponentPallette();
 
-  @DefaultMessage("Component for displaying images.  The picture to display, and other aspects of the Image\"s appearance, can be specified in the Designer or in the Blocks Editor.")
+  @DefaultMessage("Component for displaying images.  The picture to display, and other aspects of the image\"s appearance, can be specified in the Designer or in the Blocks Editor.")
```
Comment (josmas) [inline appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java]: As commented below, Image is a noun (name of the component) so the Capital is fine.

**Comment by halatmit** [comment]
Closing.   Unlclear what the right thing is, but it's minor.

**Pull request closed without merging**